tests/run.sh

```diff
@@ -179,12 +169,11 @@ if [[ -n $SPECIFIC_INTEGRATION_TEST ]]; then
 fi
 
 python2 -m pytest \
-    $XDIST_ARGS \
-    $MAX_FAIL_ARG \
-    -s \
-    --verbose \
-    --junitxml=results.xml \
-    $HTML_REPORT \
-    $pass_args \
-    $SPECIFIC_INTEGRATION_TEST_FLAG "$SPECIFIC_INTEGRATION_TEST" \
-    $DEFAULT_TESTS
+        $XDIST_ARGS \
+        $MAX_FAIL_ARG \
+        -s \
```

Comment:
The `-s` can be dropped now that we have  `--capture=no` in the ini file. (I should have done this myself in the other PR)